srml/system/src/lib.rs

```diff
@@ -404,8 +404,8 @@ impl<T: Trait> Module<T> {
 		let storage_root = T::Hashing::storage_root();
 		let storage_changes_root = T::Hashing::storage_changes_root(parent_hash, number.as_() - 1);
 
-		// we can't compute changes trie root earlier && put it to the Digest
-		// because it will include all currently existing temporaries
+		// We can't compute changes trie root earlier && put it to the Digest
```

Comment:
inconsistent with above change. to be honest, there's really no need to mess around with internal comments for algorithms. if the coder is happy with all-lowercase, then that's fine.